stable/openvpn/templates/openvpn-deployment.yaml

```diff
@@ -95,6 +95,7 @@ spec:
           {{- if .Values.openvpn.useCrl }}
           - key: "crl.pem"
             path: "crl.pem"
+            mode: 484
```

Comment:
It seems like it's possible to specify this in octal (as per https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#keytopath-v1-core), so it might be a bit more clear to write this as 0744?

Reply:
agreed, I was just following what was already there in `defaultMode` - but this can be changed in another PR I guess, have changed the new one to octal